tests/nat/agent/test_reasoning_agent.py

```
@@ -138,9 +138,11 @@ def _get_function(name: str):
 
     # get_function_dependencies is used just for referencing tool names, etc
     class FakeDeps:
-        functions = ["SomeTool"]
+        functions = {"SomeTool"}
```

Ruff (0.12.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)